apps/studio/src/server/modules/resource/resource.router.ts

```diff
@@ -265,7 +268,8 @@ export const resourceRouter = router({
               // NOTE: we only allow moves to folders/root.
               // for moves to root, we only allow this for admin
               (parent.type !== ResourceType.RootPage &&
-                parent.type !== ResourceType.Folder)
+                parent.type !== ResourceType.Folder &&
+                parent.type !== ResourceType.Collection)
```

Comment:
nit: might be good to add a comment about the frontend-side filtering, which makes moving items from Folder to Collection (and vice versa) not possible to a normal user.

Reply:
frontend side filtering removed, now added a backend side rejection + toast